lib/wallaby/experimental/selenium/webdriver_client.ex

```diff
@@ -350,24 +372,69 @@ defmodule Wallaby.Experimental.Selenium.WebdriverClient do
       do: {:ok, value}
   end
 
+  @doc """
+  Retrieves the list of window handles of all windows (or tabs) available to the session
+  """
+  @spec window_handles(parent) :: {:ok, list(String.t)}
   def window_handles(session) do
     with {:ok, resp} <- request(:get, "#{session.url}/window_handles"),
-         {:ok, value} <- Map.fetch(resp, "value"),
+          {:ok, value} <- Map.fetch(resp, "value"),
       do: {:ok, value}
   end
 
   @doc """
-  Retrieves the window handle for from session
+  Retrieves the window handle for the currently focused window (or tab) for the session
   """
-  @spec window_handle(Session.t) :: String.t
+  @spec window_handle(parent) :: {:ok, String.t}
   def window_handle(session) do
-    with  {:ok, resp} <- request(:get, "#{session.url}/window_handle"),
+    with {:ok, resp} <- request(:get, "#{session.url}/window_handle"),
           {:ok, value} <- Map.fetch(resp, "value"),
-      do: value
+      do: {:ok, value}
+  end
+
+  @doc """
+  Changes focus to another window (or tab)
+
+  The window to change focus to may be specified by its server assigned
+  window handle, or by the value of its name attribute.
```

Comment:
"You may specify the the window by its server-assigned window handle or by the value of its name attribute."

WDYT?